clipper_admin/clipper_manager.py

```diff
@@ -735,6 +744,105 @@ def centered_predict(inputs):
 
         return deploy_result
 
+    def _register_app_and_check_success(self, name, input_type, default_output,
+                                        slo_micros):
+        if self.register_application(name, name, input_type, default_output,
+                                     slo_micros):
+            print("Application registration sucessful! Deploying model.")
+            return True
+        print("Application registration unsuccessful. Will not deploy model.")
+        return False
+
+    def activate_predict_function(self,
```

Comment:
Activate isn't really the right verb. I can't think of a good shorter name that doesn't conflict with our existing API, so maybe just use the long names from before.